### ISSUE TITLE
Traverse symlinked directories within tilesets directory

### ISSUE THREAD

**Issue by brendan-ward** [consbio/mbtileserver#93]
Right now, it seems like we are not traversing tilesets that are in symlinked directories within the tileset directory.

This would allow symlinking multiple subdirectories into a single tileset directory provided to the server.

**Comment by asmyasnikov** [comment]
In mounted volume in docker container symlinks to out from volume may be broken =(

**Comment by brendan-ward** [comment]
@asmyasnikov is the same true when you provide multiple root directories to Docker, as per #92?  I would expect that you'd need to mount host directories as volumes for your Docker container.

**Comment by asmyasnikov** [comment]
Yes, you right. I will mount each directory separately as volumes. But if feature of multiple paths you not approved, I must be use symlinks. Good symlinks inside docker may be bad outside. And vice versa.
